### PR TITLE
Limit memory used by redis

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: geoportal/c2cgeoportal_geoportal/scaffolds/create/docker-compose.yaml.mako [hunk 1]
```diff
@@ -43,6 +43,9 @@ ${service_defaults('memcached', 11211)}\
 
   redis:
     image: redis:3.2
+    command:
+      - redis-server
+      - /etc/redis/redis.conf
```
Comment (pvalsecc) [inline geoportal/c2cgeoportal_geoportal/scaffolds/create/docker-compose.yaml.mako]: Don't you have a `volumes-from` to add?

File: geoportal/c2cgeoportal_geoportal/scaffolds/create/redis/redis.conf.tmpl [hunk 2]
```diff
@@ -0,0 +1,2 @@
+maxmemory ${REDIS_MAXMEMORY}
+maxmemory-policy allkeys-lru
```
Comment (pvalsecc) [inline geoportal/c2cgeoportal_geoportal/scaffolds/create/redis/redis.conf.tmpl]: You can pass that from the command line: https://redis.io/topics/config

And I recommend adding: `--save "" --appendonly no`

File: geoportal/c2cgeoportal_geoportal/scaffolds/create/docker-compose.yaml.mako (hunk 1 above)
Comment (sbrunner) [reply geoportal/c2cgeoportal_geoportal/scaffolds/create/docker-compose.yaml.mako]: +1 :-)

File: geoportal/c2cgeoportal_geoportal/scaffolds/create/redis/redis.conf.tmpl (hunk 2 above)
Comment (pvalsecc) [reply geoportal/c2cgeoportal_geoportal/scaffolds/create/redis/redis.conf.tmpl]: https://github.com/camptocamp/private-geo-charts/blob/177b618cc5ba9f7fab8e6209fd2ae204442dcc97/c2cgeoportal/templates/redis-deployment.yaml#L34-L42

Comment (sbrunner) [reply geoportal/c2cgeoportal_geoportal/scaffolds/create/redis/redis.conf.tmpl]: Thanks, fixed :-)

File: geoportal/c2cgeoportal_geoportal/scaffolds/create/docker-compose.yaml.mako [hunk 3]
```diff
@@ -43,6 +43,16 @@ ${service_defaults('memcached', 11211)}\
 
   redis:
     image: redis:3.2
```
Comment (pvalsecc) [inline geoportal/c2cgeoportal_geoportal/scaffolds/create/docker-compose.yaml.mako]: Why not redis:4?

**Pull request merged**